src/services/participantsService.js

```diff
@@ -252,6 +253,10 @@ const getUserAbsence = async (userId) => {
 	return axios.get(generateOcsUrl('/apps/dav/api/v1/outOfOffice/{userId}/now', { userId }))
 }
 
+const getReplacementUser = async (userId) => {
+	return axios.post(generateUrl('contactsmenu/findOne'), `shareType=0&shareWith=${userId}`)
```

Comment:
This is restricted by sharing restrictions.
Can't we simply return the user ID and display name on the ooo endpoint?

Reply:
added